packages/tools/pluggable-widgets-tools/tests/commands.js

```diff
@@ -231,6 +234,15 @@ async function main() {
             }
         }
 
+        async function checkDependenciesFiles() {
+            if (
+                !existsSync(join(workDir, "dist", "tmp", "widgets", "dependencies.json")) ||
+                !existsSync(join(workDir, "dist", "tmp", "widgets", "dependencies.txt"))
```

Comment:
It would be nice to also check the content of the file, to make sure it matches that which is expected

Reply:
done

Reply:
done 👍